lib/util.js

```diff
@@ -209,7 +213,7 @@ function callbackify(original) {
     // implications (stack, `uncaughtException`, `async_hooks`)
     ReflectApply(original, this, args)
       .then((ret) => process.nextTick(cb, null, ret),
-            (rej) => process.nextTick(callbackifyOnRejected, rej, cb));
+        (rej) => process.nextTick(callbackifyOnRejected, rej, cb));
```

Comment:
It seems there are a few of these unrelated changes?

Reply:
I ran lint fix, everything is back to how it should be.